infra-templates/ipsec/16/rancher-compose.yml

```diff
@@ -69,7 +81,12 @@
       type: "string"
       default: '1h'
       required: true
-
+    - variable: "RANCHER_IPSEC_PSK"
+      label: "PSK for IPSec"
+      description: "Proceed with caution; typical users will not need to change this. Incorrect values can break your Rancher installation."
```

Comment:
Let's reword to: `This is an advanced setting and most users will not need to use this. Incorrect values can break your Rancher installation.`